examples/studio.html

```diff
@@ -905,6 +905,14 @@
           viewport.set(m.data.viewport[0], 1 - m.data.viewport[1] - m.data.viewport[3], m.data.viewport[2], m.data.viewport[3]);
           break;
         }
+        case 'exit': {
+          context.destroy();
+          window.document.resources.addEventListener('drain', () => {
+            console.log('drain');
+            process.exit();
```

Comment:
`process` should really not be available here. If it is, that is a separate problem.

Reply:
Oh no, process definitely isn't available. I just noticed I crashed my way to an exit here.

Reply:
Phew!